MonsterHunterStories2/Monster.cs

```diff
@@ -239,5 +239,10 @@ public uint DragonElementDefenseBonus
 			get { return SaveData.Instance().ReadNumber(mAddress + 180, 1); }
 			set { Util.WriteNumber(mAddress + 180, 1, value, 0, 0xFF); }
 		}
+		public uint KinshipGauge
+		{
+			get { return SaveData.Instance().ReadNumber(mAddress + 400, 1); }
+			set { Util.WriteNumber(mAddress + 400, 1, value, 0, 100); }
```

Comment:
This value is actually "battles won" (essentially how many times it was used in a battle you won - not counting when it was sidelined the whole fight) and affects starting kinship. It effectively caps at 30 (0x1E) as that's the most starting kinship a monstie will ever have - though the stored value will easily climb higher than that.

8 bytes before it is "lifetime battles" or all battles its been part of the party for - whether it was actually used or not. the higher this number the "smarter' the monstie will act (read as: the more 'battle experience' it has, the less likely it is to pick a move that loses a head-to-head)